src/uct/ib/ud/base/ud_ep.h

```diff
@@ -191,20 +194,22 @@ enum {
     UCT_UD_EP_FLAG_PRIVATE           = UCS_BIT(1), /* EP was created as internal */
     UCT_UD_EP_FLAG_HAS_PENDING       = UCS_BIT(2), /* EP has some pending requests */
     UCT_UD_EP_FLAG_CONNECTED         = UCS_BIT(3), /* EP was connected to the peer */
+    UCT_UD_EP_FLAG_ON_CEP            = UCS_BIT(4), /* EP was inserted to connection
```

Comment:
seems like this is a debug flag?

Reply:
@yosefe reverted the change, it's not a debug flag, since we use it when removing EP from CEP - checking whether the EP is on CEP or not